packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -240,6 +240,11 @@ export class RunningSummarizer implements IDisposable {
         await this.summarizingLock;
     }
 
+    public async stopOnDemand() {
```

Comment:
Why do we need this (public) API? Can you please put some comments describing when and how this function can be called, and what impact it has on the rest of the system (including how currently running summary fails, if it does)